modules/troubleshooting/pages/index.adoc

```diff
@@ -318,6 +318,34 @@ linking and services:: See the `skupper-service-controller-*-events.txt` file to
 
 --
 
+// Type:procedure
+[id='network-performance']
+== Improving network performance
+
+If you encounter network performance issues, you can scale Skupper to address those concerns.
+For example, use this procedure to increase network throughput, and if you have many clients, network latency.
```

Comment:
I would maybe drop the word 'network' from this paragraph.

Reply:
>If you encounter performance issues, you can scale Skupper to address those concerns.

@grs How would I spot 'performance issues' with Skupper?

Reply:
I've tried to address your point, using 'Skupper router performance'